mmocr/utils/ocr.py

```diff
@@ -400,6 +420,99 @@ def __init__(self,
             if hasattr(model, 'module'):
                 model = model.module
 
+    @staticmethod
+    def tesseract_det_inference(imgs):
+        """Inference image(s) with the tesseract detector.
+
+        Args:
+            imgs (list[ndarray]): images to inference.
+
+        Returns:
+            result (dict): Predicted results.
+        """
+        assert is_type_list(imgs, np.ndarray)
+
+        # Very very gross way to find tessdata :(
+        if sys.platform == 'linux':
+            api = PyTessBaseAPI()
+        elif sys.platform == 'win32':
+            try:
+                p = subprocess.Popen(
+                    'where tesseract', stdout=subprocess.PIPE, shell=True)
+                s = p.communicate()[0].decode('utf-8').split('\\')
+                path = s[:-1] + ['tessdata']
+                tessdata_path = '/'.join(path)
+                api = PyTessBaseAPI(path=tessdata_path)
+            except RuntimeError:
+                raise RuntimeError('Please install tesseract first. '
+                                   'Check out the installation guide at'
+                                   'https://github.com/sirfz/tesserocr')
+        else:
+            raise NotImplementedError
```

Comment:
We can wrap this into a method `MMOCR.get_tesseract_api()` so that it can be reused by det and recog inference.